### DISCUSSION TITLE
Modify road and multi view camera in ULC 4.0 @ - Duster 2

### DISCUSSION THREAD

**Discussion by BBFul** [cedricp/ddt4all#1048] [Q&A]
Hello! I ve changed my medianav which was 9.1.3 with 1.0.15.0 version. I want to config road and multiview camera but it won t allow me to do that. It appear 7F2E22 error. I ve changed with succes other settings like view clim and temp in display. What can I do to change these settings? Thanks!
![20240309_133938](https://github.com/cedricp/ddt4all/assets/55613397/0906f4b2-32ce-4954-bdaf-104819bf08d1)

**Comment by shrlnm** [comment]
Hello. Which specific xml file did you use and which parameter did you change?

**Comment by BBFul** [comment]
Hello, shrlnm! tried to change road and multi view camera from drom desactivated/not fitted tp activated/fitted without success.
![Screenshot_12](https://github.com/cedricp/ddt4all/assets/55613397/537c55a6-110e-4b37-a1d2-b52020bf0dc9)
![Screenshot_8](https://github.com/cedricp/ddt4all/assets/55613397/c8b256cf-c81f-4e73-8450-b1320f092a94)

**Reply by shrlnm** [reply]
@BBFul The ROAD and MVC are absent on X79 platform
On XJD these functions are enabled  with such MTC formula
"XJD/RAD14D,RA42D,RAD15D,RA43D/TR4X4/ ESPDES,ESPHSA"
and
MVCAM

**Reply by BBFul** [reply]
Shrlnm, thanks!

**Reply by KarelSvo** [reply]
[clic](https://www.dacianer.de/threads/meine-aenderungen-am-lodgy.37298/page-2)
